fabric_client/README.md

```diff
@@ -0,0 +1,48 @@
+# Switchtec Fabric DMA Driver Client Examples
+
+## Summary
+
+There are two Switchtec fabric DMA clients in this directory for people to
+understand the proprietary fabric APIs of Switchtec DMA driver. They work in
+the initiator and target mode, which is considered the most common use case of
```

Comment:
They work in initiator and target mode, which are considered the most common use cases of Switchtec fabric DMA.